content/blog/3-same-subject-continued-concerning-dangers/index.md

```diff
@@ -1,45 +1,40 @@
 
 In order to have peace, we need to be sure to observe international laws and norms with regard to these powerful countries, and it seems obvious to me that this will be easier and more effective if done by a single national government rather than by thirteen separate states or a few confederacies.
 
-Once an efficient national government is established, the best people in the country will naturally serve there. Because we can select from the entire country, we will never have the problem that some states have where they can't really find anyone qualified and the cream of the crop will serve in the national government <sup>[1](#footnote1)</sup>. The administration, political bodies, and judicial decisions of a national government will, therefore, be more wise, well-reasoned, and judicious than those of individual states. Having this group of our best people in charge of our national direction will lead to better relations with other countries, and therefore more safety for us.
+Once an efficient national government is established, the best people in the country will naturally serve there. Because we can select from the entire country, we will never have the problem that some states have where they can't really find anyone qualified. Rather, we will see the cream of the crop join the national government. The administrative, political, and judicial decisions of a national government will, therefore, be more wise, well-reasoned, and judicious than those of individual states. Having this group of our best people in charge of our national direction will lead to better relations with other countries, and therefore more safety for us.
 
 Our relationship with these other countries will require making and executing treaties. Can you imagine trying to enforce treaties with thirteen different, independent court systems, with judges appointed by different, independent governments! 🤮 The members of the Constitutional Convention were wise to commit these sorts of questions to a single, national court.
 
-Indeed, sometimes the governing party of a single state might be tempted to gain an advantage by breaking a treaty. However, a single state won't have enough power to cause a national government to make these sorts of bad decisions, and justice will be preserved. See the peace treaty with Britain for an example of how important this is. <!-- Worth looking into what he's talking about -->
+Indeed, sometimes the governing party of a single state might be tempted to gain an advantage by breaking a treaty, bringing war to all of us. <!--This clause is commentary, really, and not in the original--> However, a single state wouldn't have enough power to force a national government to make these sorts of bad decisions, and justice will be preserved. See the peace treaty with Britain for an example of how important this is. <!-- Worth looking into what he's talking about -->
```

Comment:
I had to read this a few times to understand it. Simpler language might be helpful. Something like:

> A single state might be tempted to break a treaty, but with a strong federal government, no one state has enough power to make that decision and bring war to us all.

Reply:
I also think we should either call out explicitly which state broke (or tried to break) the treaty with Britain, and what happened, OR we should cut the last sentence.